pool/helpers.go

```diff
@@ -33,7 +33,13 @@ func executeJob(ctx context.Context, id string, j amboy.Job, q amboy.Queue) {
 		jobCtx = ctx
 	}
 	j.Run(jobCtx)
-	q.Complete(ctx, j)
+	if err := q.Complete(ctx, j); err != nil {
+		grip.Error(message.WrapError(err, message.Fields{
+			"message":  "could not mark job as complete",
```

Comment:
Small nit: can you change this and L113 to `could not mark job complete` to be consistent with all the other complete error messages?